lib/repoaudit.rb

```diff
@@ -0,0 +1,14 @@
+require 'bundler/setup'
+require 'github_api'
+require 'awesome_print'
+require 'pp'
+require 'pry-byebug'
+require 'open-uri'
+
+module Repoaudit
```

Comment:
Can this be `RepoAudit`? Otherwise it's a little unclear about how it's pronounced…